app/src/main/res/layout/random_preset_fragment.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  tools:context=".fragment.RandomPresetFragment">
+
+  <LinearLayout
+    app:layout_constraintTop_toTopOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingStart="10dp"
+    android:paddingEnd="10dp">
+
+    <TextView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="7dp"
+      android:text="@string/mood"
+      android:textAppearance="?attr/textAppearanceHeadline5"
+      android:layout_marginTop="10dp"/>
+
+    <RadioGroup
+      android:id="@+id/preset_type"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:checkedButton="@id/preset_type__any"
+      android:layout_marginStart="10dp">
+
+      <RadioButton
+        android:id="@+id/preset_type__any"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/any" />
+
+      <RadioButton
+        android:id="@+id/preset_type__focus"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/focus" />
+
+      <RadioButton
+        android:id="@+id/preset_type__relax"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/relax" />
+
+    </RadioGroup>
+
+    <TextView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="20dp"
+      android:layout_marginBottom="7dp"
+      android:text="@string/intensity"
+      android:textAppearance="?attr/textAppearanceHeadline5" />
+
+    <TextView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginVertical="7dp"
+      android:text="@string/intensity_description" />
```

Comment:
Move this to the top of the layout. And change the message. The message could be: _Generated random presets will contain sounds corresponding to the chosen mood. Moreover, dense presets will attempt to play more sounds simultaneously._